duo_client/admin.py

```diff
@@ -237,11 +254,19 @@ def get_administrator_log(self,
             row['host'] = self.host
         return response
 
-    def get_authentication_log(self,
-                               mintime=0):
+    def get_authentication_log(self, api_version=1, **kwargs):
```

Comment:
This signature change will break any existing clients that invoked this method with `mintime` as a positional argument.

Accepting this change means the next release of duo_client_python is not backwards compatible. Perhaps the change can be re-written in a way to make it backwards compatible. (There may be other changes already pulled that break backwards compatibility? In which case, ignore me!)